src/python/pants/engine/subsystem/native_engine_version

```diff
@@ -1 +1 @@
-ae3b1ec1714da0a43486be8dda37cdb8aeca815b
+ea591fc0b0c25f189f18ea8fda225526bd389503
```

Comment:
Actually, these continue to need to be committed: you'll need to merge from master, and re-run your build to commit some 3rd SHA that will be the union of Kris' change and your change.